uda_udf_test.py

```diff
@@ -0,0 +1,111 @@
+from typing import Tuple, NamedTuple, Callable
+
+from sdcm.cluster import BaseNode
+from sdcm.stress_thread import CassandraStressThread
+from sdcm.tester import ClusterTester
+from sdcm.utils.udf import UDFS
+from sdcm.utils.uda import UDAS
+
+
+class UDFVerification(NamedTuple):
+    name: str
+    query: str
+    verifier_func: Callable
+
+
+class UDAUDFTest(ClusterTester):
+    """
+    Test Scylla with User Defined Functions and User Defined Aggregates,
+    using cassandra-stress.
+    """
+    KEYSPACE_NAME = "ks"
+    CF_NAME = "uda_udf"
+
+    def test_uda_and_udf(self) -> None:
+        self.log.info("Starting UDA/UDF test...")
+        self.prewrite_db_with_data()
+        node: BaseNode = self.db_cluster.get_node()
+
+        for udf in UDFS.values():
+            self.log.info("Creating the following UDF: %s", udf.name)
+            cmd = udf.get_create_query(ks="ks")
+            with self.db_cluster.cql_connection_patient(node=node) as session:
+                session.execute(cmd)
+            self.log.info("UDF %s created", udf.name)
+
+        self._verify_udf_functions()
+
+        for uda in UDAS.values():
```

Comment:
please add log.info that UDA is created similary to UDF for better understanding.
Also, why there's no UDA verification like in UDF?

Reply:
Added. And now there is a UDA verification, that's a good idea, I originally missed.